test/unit/model/result/ResultServiceTest.php

```diff
@@ -116,15 +116,15 @@ public function testProcessReplaceRequest()
         $this->assertNotSame($response->getMessageIdentifier(), $response->getMessageRefIdentifier());
         $this->assertSame('msg_identifier', $response->getMessageRefIdentifier());
         $this->assertSame('replaceResultRequest', $response->getOperationRefIdentifier());
-        $this->assertContains('de_id', $response->getStatusDescription());
-        $this->assertContains('0.234', $response->getStatusDescription());
+        $this->assertStringContainsString('de_id', $response->getStatusDescription());
+        $this->assertStringContainsString('0.234', $response->getStatusDescription());
```

Comment:
Why was this change necessary?

Reply:
https://github.com/oat-sa/extension-tao-lti-consumer/pull/47#discussion_r488559151

Reply:
thanks for the answer :)